agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4479,8 +4479,9 @@ bool slave_thread::handle_autoconfiguration_wsc(Socket *sd, ieee1905_1::CmduMess
 
     while ((type = cmdu_rx.getNextTlvType()) != int(ieee1905_1::eTlvType::TLV_END_OF_MESSAGE)) {
         if (type == int(wfa_map::eTlvTypeMap::TLV_AP_RADIO_IDENTIFIER)) {
+            if ((ruid = cmdu_rx.addClass<wfa_map::tlvApRadioIdentifier>()) == nullptr)
```

Comment:
smart pointers should be checked with '!' and not be compared to nullptr. Comparing to nullptr will cause klocwork issues.
use `if (!ruid) `instead.